<launch>
  <node pkg="pure_pursuit" type="pure_pursuit" name="pure_pursuit" output="screen">
    <remap from="~input/reference_trajectory" to="/planning/scenario_planning/trajectory"/>
    <remap from="~input/current_velocity" to="/localization/twist"/>
    <remap from="~output/control_raw" to="lateral/control_cmd"/>

    <!-- Node Parameters -->
    <param name="control_period" value="0.033"/>

    <!-- Algorithm Parameters -->
    <param name="lookahead_distance_ratio" value="2.2"/>
    <param name="min_lookahead_distance" value="2.5"/>
    <param name="reverse_min_lookahead_distance" value="7.0"/>
  </node>
</launch>
